src/V3LinkCells.cpp

```diff
@@ -105,6 +109,7 @@ class LinkCellsVisitor final : public VNVisitor {
 
     // STATE
     VInFilter* const m_filterp;  // Parser filter
+    LinkCellsCtx* const m_ctx;  // Context for linking cells
```

Comment:
Might as well pass as reference to constructor and into here.

```suggestion
    LinkCellsState& const m_stater;  // State for linking cells
```